src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.search-form {
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  margin-top: 20px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.search-form:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-form:focus {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.form-size-big {
+  height: 70px;
+  background-position: 25px 24px;
+  background-size: 19px 19px;
+}
+
+.form-size-small {
+  height: 42px;
+  background-position: 12px 14px;
+  background-size: 11px 11px;
+}
+
+.search-bar {
+  height: 100%;
+  width: 100%;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-left: 0;
+  border-right: 0;
+  border-radius: 4px;
+  padding: 0;
+  font-family: inherit;
+  color: inherit;
+}
+
+.search-bar__big-form {
+  margin-left: 62px;
+  font-size: 16px;
+}
+
+.search-bar__big-form:focus {
+  font-weight: 900;
+  color: #3d4e61;
```

Comment:
Consider will be better if you create a variable for repeating colors and use it everywhere